### DISCUSSION TITLE
How to use python package with HDFS support

### DISCUSSION THREAD

**Discussion by momhiar** [delta-io/delta-rs#1577] [Q&A]
Hi,
whenever i want to use python with HDFS it throws :

`Delta-rs must be build with feature 'hdfs' to support loading from: hdfs://localhost:9000/home/hduser/`

code: 
```python 
from deltalake import DeltaTable

dt = DeltaTable("hdfs://localhost:9000/home/hduser/directory")
dt.version()
dt.files()
```

deltalake pypi package versions tested: 
0.10.0
0.10.1

**Comment by roeap** [comment]
We have been discussing this a bit and essentailly decided to not include hdfs support in the python bindings, as that would mean we would always have a dependency on java, needed or not ...

In case you are willing to build the python bindings from source, you could however activate the hdfs feature in the python bindings, at which point it should work ...
